### PR TITLE
[PKG-3011] textstat 0.7.3 :snowflake:

### PR DESCRIPTION
**Pull request by skupr-anaconda**
**Current package info**

| Channel | Downloads | Version | Platforms | License |
| --- | --- | --- | --- | --- |
| conda-forge | [Conda Downloads](https://anaconda.org/conda-forge/textstat) | [Conda Version](https://anaconda.org/conda-forge/textstat) | [Conda Platforms](https://anaconda.org/conda-forge/textstat) | Conda License |

Changelog: https://github.com/textstat/textstat/releases
License: https://github.com/textstat/textstat/blob/main/LICENSE
Requirements:
- https://github.com/textstat/textstat/blob/0.7.3/requirements.txt
- https://github.com/textstat/textstat/blob/0.7.3/setup.py

Actions:
1.

### CODE REVIEW

File: abs.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+upload_channels:
+  - https://staging.continuum.io/prefect/fs/pyphen-feedstock/pr2/25bf479
```
Comment (lorepirri) [inline abs.yaml]: when pyphen is done:

```suggestion
upload_channels:
  - sfe1ed40
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,39 +1,46 @@
...
-    - setuptools
+    - python
 
 test:
   imports:
     - textstat
+  requires:
+    - pip
+  commands:
+    - pip check
```
Comment (lorepirri) [inline recipe/meta.yaml:33]: we could add tests. I did not execute this but it should work:

```suggestion
  requires:
    - pip
    - pytest
  source_files:
    - test.py
  commands:
    - pip check
    - pytest test.py
```

**Review by lorepirri — APPROVED** [review]
LGTM, remember the upload channel, and maybe add tests

File: recipe/meta.yaml (hunk 2 above)
Comment (M-Waszkiewicz-Anaconda) [reply recipe/meta.yaml:33]: yes, that is good idea

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
looks good, maybe try to add tests if not much work

File: recipe/meta.yaml (hunk 2 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml:33]: `test.py` isn't available inside the Source Distribution https://pypi.org/project/textstat/0.7.3/#files

Comment (lorepirri) [reply recipe/meta.yaml:33]: 😞

**Comment by skupr-anaconda** [comment]
An error on `linux-aarch64` after merging the PR:
```
conda.exceptions.ResolvePackageNotFound:
  - pyphen
```
It's weird because `pyphen` is available on `linux-aarch64` in defaults:
```
https://repo.anaconda.com/pkgs/main/linux-aarch64
pyphen-0.14.0-py310hd43f75c_0.tar.bz2
pyphen-0.14.0-py311hd43f75c_0.tar.bz2
pyphen-0.14.0-py38hd43f75c_0.tar.bz2
pyphen-0.14.0-py39hd43f75c_0.tar.bz2
```

I've restarted a flow on Prefect

**Pull request merged**